packages/eslint-plugin-edu-react/src/index.ts

```diff
@@ -7,6 +13,7 @@ module.exports = {
     },
   },
   rules: {
+    '@chanzuckerberg/eslint-plugin-edu-react/use-effect-deps-presence': 'error',
```

Comment:
Maybe this should be

```suggestion
    '@chanzuckerberg/eslint-plugin-edu-react/use-effect-deps-presence': 'warning',
```

🤔